### ISSUE TITLE
Mute Button

### ISSUE THREAD

**Issue by Xanture** [austriker27/ghostown#70]
On the home page, the mute button is a play symble. If you mute, then select the ghost and continue. On the next page the music is playing again and now to mute it you must press the pause symbol.

**Comment by austriker27** [comment]
Yeah we looked into that a few days ago and we were able to fix it by removing autoplay. Although that caused another issue from a UX perspective so we left it as is.

Leaving issue open for further research.
